<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fff;
color :#000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#fff7b3;
color :#000</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Conectar Porta Serial</string>
     </attribute>
     <widget class="QPushButton" name="btn_okPortaSerial">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>309</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Informe o nome da porta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_portaSerial">
         <property name="styleSheet">
          <string notr="true">background-color:#fff;
color :#000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cadastrar</string>
     </attribute>
     <widget class="QLabel" name="lbl_leitura">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Inserir">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>323</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_nomeObjeto">
         <property name="font">
          <font>
           <family>Source Code Pro</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Informe o nome do objeto: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_nomeObjeto">
         <property name="font">
          <font/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#fff;
color :#000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Emprestimos</string>
     </attribute>
     <widget class="QTableWidget" name="tabelaEmprestimo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>501</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>165</number>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data do Empréstimo</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lbl_msgEmprestimo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_emprestimo">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>441</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_nomeEmprestimo">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="le_nomeEmprestimo">
           <property name="styleSheet">
            <string notr="true">background-color:#fff;
color :#000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_dataEmprestimo">
           <property name="font">
            <font>
             <family>Source Code Pro</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Data do Empréstimo:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="de_emprestimo">
           <property name="styleSheet">
            <string notr="true">background-color:#fff;
color :#000</string>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2019</year>
             <month>1</month>
             <day>20</day>
            </datetime>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2030</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2019</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="date">
            <date>
             <year>2019</year>
             <month>1</month>
             <day>20</day>
            </date>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tabela de Controle</string>
     </attribute>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>501</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#e7e7e7;
color :#000</string>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>165</number>
      </attribute>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Código do Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disponibilidade</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo_Cadastro">
   <property name="text">
    <string>Novo Cadastro</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
